src/features/harborView/forms/berthForm/__tests__/__snapshots__/BerthEditForm.test.jsx.snap

```diff
@@ -2,4 +2,4 @@
text\\" value=\\"1\\"></div></div></div><hr><div class=\\"Dropdown-module_root__3ro8_ select\\"><label for=\\"mooringType-toggle-button\\" class=\\"FieldLabel-module_label__2tl-4 text-input_hds-text-input__label__pDw_W \\" id=\\"mooringType-label\\">Kiinnitystapa<span aria-hidden=\\"true\\" class=\\"RequiredIndicator-module_indicator__Pt7GX text-input_hds-text-input__required__2K-Bs\\">*</span></label><div class=\\"Dropdown-module_wrapper__1sFxz\\"><button type=\\"button\\" id=\\"mooringType-toggle-button\\" aria-haspopup=\\"listbox\\" aria-expanded=\\"false\\" aria-labelledby=\\"mooringType-label mooringType-toggle-button\\" class=\\"Dropdown-module_buttonDropdown__2xA1D Dropdown-module_dropdown__34HBV\\"><span class=\\"Dropdown-module_buttonLabel__3y4eg\\">Aisapaikka</span><svg xmlns=\\"http://www.w3.org/2000/svg\\" class=\\"Icon-module_icon__1Jtzj icon_hds-icon__1YqNC Icon-module_s__2WGWe icon_hds-icon--size-s__2Lkik Dropdown-module_angleIcon__3-f8N\\" viewBox=\\"0 0 24 24\\" role=\\"img\\"><g fill=\\"none\\" fill-rule=\\"evenodd\\"><path d=\\"M0 0h24v24H0z\\"></path><path fill=\\"currentColor\\" d=\\"M12 13.5l5-5 1.5 1.5-6.5 6.5L5.5 10 7 8.5z\\"></path></g></svg></button></div><ul id=\\"mooringType-menu\\" role=\\"listbox\\" aria-labelledby=\\"mooringType-label\\" tabindex=\\"-1\\" class=\\"Dropdown-module_menu__3xwA0 Dropdown-module_overflow__3tPgF\\" style=\\"max-height: calc(var(--dropdown-height) * 5);\\"></ul></div><div class=\\"TextInput-module_root__2CMNr text-input_hds-text-input__2LODq\\"><label for=\\"comment\\" class=\\"FieldLabel-module_label__2tl-4 text-input_hds-text-input__label__pDw_W \\">Huomiot</label><div class=\\"TextInput-module_inputWrapper__3Rvel text-input_hds-text-input__input-wrapper__1OqYG\\"><input class=\\"TextInput-module_input__1BlHi text-input_hds-text-input__input__GJm5C\\" id=\\"comment\\" type=\\"text\\" value=\\"\\"></div></div><div class=\\"Checkbox-module_checkbox__3L0GR checkbox_hds-checkbox__9HMCz checkbox\\"><input id=\\"isActive\\" class=\\"Checkbox-module_input__3VZvy checkbox_hds-checkbox__input__1w0pu\\" type=\\"checkbox\\" value=\\"\\" checked=\\"\\"><label for=\\"isActive\\" class=\\"Checkbox-module_label__L5AN1 checkbox_hds-checkbox__label__3HoD3\\">Paikka on käytössä</label></div><div class=\\"formActionButtons\\"><button type=\\"button\\" class=\\"Button-module_button__3wgee button_hds-button__2A0je Button-module_secondary__399Vd button_hds-button--secondary__1NOWS Button-module_theme-coat__2xA5D button_hds-button--theme-coat__12cO0\\" color=\\"supplementary\\"><span class=\\"Button-module_label__2zMLC button_hds-button__label__2EQa-\\">Peruuta</span></button><button type=\\"submit\\" class=\\"Button-module_button__3wgee button_hds-button__2A0je Button-module_primary__3bo_T button_hds-button--primary__2NVvO Button-module_theme-coat__2xA5D button_hds-button--theme-coat__12cO0\\"><span class=\\"Button-module_label__2zMLC button_hds-button__label__2EQa-\\">Tallenna</span></button></div></form>"`;
```

Comment:
This is another mega snapshot, should we `.render()` it?

Reply:
Done